build.sh

```diff
@@ -1,18 +1,17 @@
 #!/bin/bash 
-BUILD_ENV="1.0.6.1"
+BUILD_ENV="1.0.7"
 SCRIPT_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
 if [ ! -d "build-env" ]; then
     mkdir build-env || exit 1
 fi
 cd build-env || exit 2
 
-if [ ! -f "PF-build-env-Linux64-$BUILD_ENV.zip" ]; then
-    #wget https://github.com/3d-gussner/PF-build-env-1/releases/download/$BUILD_ENV-Linux64/PF-build-env-Linux64-$BUILD_ENV.zip || exit 3
-	wget https://github.com/prusa3d/PF-build-env/releases/download/$BUILD_ENV-Linux64/PF-build-env-Linux64-$BUILD_ENV.zip || exit 3
-fi
-
 if [ ! -d "../../PF-build-env-$BUILD_ENV" ]; then
+    if [ ! -f "PF-build-env-Linux64-$BUILD_ENV.zip" ]; then
+        wget https://github.com/3d-gussner/PF-build-env-1/releases/download/$BUILD_ENV-Linux64/PF-build-env-Linux64-$BUILD_ENV.zip || exit 3
+        #wget https://github.com/prusa3d/PF-build-env/releases/download/$BUILD_ENV-Linux64/PF-build-env-Linux64-$BUILD_ENV.zip || exit 3
```

Comment:
Why it is not new build-env got from prusa3d repository?

Reply:
The URLs will be changed to prusa3d repos and branches as soon the Arduino_boards and PF-build-env are approved and merged. 
I am using my forks and branches for testing.